config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-presubmits-release-1-0.yaml

```diff
@@ -0,0 +1,152 @@
+preubmits:
```

Comment:
those jobs for the release-1.0 might fail because we will need to cherry-pick the changes we will make in the main branch to the release 1.0 branch. I would say we drop those things for now from this PR make the main branch work and then cherry pick the changes to the release branch and then make a new PR to add the jobs